install-pre-commit-hook.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Link pre-commit hook for a repo in the directory supplied as an argument, or the parent repo of this file.
+#
+# USAGES:
+# $ /path/to/install-pre-commit-hook.sh
+# $ /path/to/install-pre-commit-hook.sh wp-content/themes/my-theme
+
+set -e
+
+# Store the initial directory to return back to it later
+OLDPWD=`pwd`
```

Comment:
I think we should use `$()` for command substitution in this script instead of backticks. See http://mywiki.wooledge.org/BashFAQ/082